swagger.yaml

```diff
@@ -657,6 +657,55 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/progress:
+    put:
+      tags:
+        - installer
+      security:
+        - agentAuth: []
+      summary: Update cluster installation progress.
+      operationId: UpdateClusterInstallProgress
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The ID of the cluster to retrieve.
+          type: string
+          format: uuid
+          required: true
+        - in: body
+          name: cluster-progress
+          description: Cluster install progress value.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
```

Comment:
If there is nothing to return, the response code should be 204 and not 200.

Reply:
Done.